tests/e2e/default/enketo/enketo-widgets.wdio-spec.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const placeFactory = require('@factories/cht/contacts/place');
+const userFactory = require('@factories/cht/users/users');
+const utils = require('@utils');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const enketoWidgetsPage = require('@page-objects/default/enketo/enketo-widgets.wdio.page');
+const genericForm = require('@page-objects/default/enketo/generic-form.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+
+
+describe('Enketo Widgets', () => {
+  const districtHospital = placeFactory.place().build({ _id: 'dist1', type: 'district_hospital' });
+  const offlineUser = userFactory.build({ place: districtHospital._id, roles: ['chw'] });
+  const formDoc = {
+    _id: 'form:enketo_widgets_test',
+    internalId: 'enketo_widgets_test',
+    title: 'Enketo Widgets Test',
+    type: 'form',
+    _attachments: {
+      xml: {
+        content_type: 'application/octet-stream',
+        data: Buffer
+          .from(fs.readFileSync(`${__dirname}/forms/enketo_widgets.xml`, 'utf8'))
+          .toString('base64'),
+      },
+    },
+  };
+
+  before(async () => {
+    await utils.saveDoc(formDoc);
+    await utils.saveDocs([districtHospital]);
```

Comment:
```suggestion
    await utils.saveDocs([formDoc, districtHospital]);
```
I couldn't test this locally before logging off today but I will try again first thing tomorrow morning and report back if we can refactor it this way

Reply:
You are completely right, I just tested is and it worked! Thank you ⭐